sunkit_spec/sunxspex_fitting/tests/test_fitter.py

```diff
@@ -4,8 +4,8 @@
 import pytest
 from numpy.testing import assert_allclose
 
-from sunxspex.sunxspex_fitting.fitter import SunXspex
-from sunxspex.sunxspex_fitting.instruments import CustomLoader, InstrumentBlueprint
+from sunkit_spec.sunxspex_fitting.fitter import sunkit_spec
```

Comment:
this may be confusing, and an issue with namespace?

we should probably rename the class to something else - any suggestions?

Reply:
If you mean `sunxspex_fitting` then I agree! Maybe even just `fitting` would do?

Reply:
Yea I don't want to change too much in his PR also kinda want to leave `sunlit_spex.fitting` free for the refactor.